internal/ansible/controller/reconcile.go

```diff
@@ -273,20 +273,25 @@ func (r *AnsibleOperatorReconciler) Reconcile(ctx context.Context, request recon
 	return reconcileResult, nil
 }
 
-func printEventStats(statusEvent eventapi.StatusJobEvent) {
+func printEventStats(statusEvent eventapi.StatusJobEvent, u *unstructured.Unstructured) {
 	if len(statusEvent.StdOut) > 0 {
-		fmt.Printf("\n--------------------------- Ansible Task Status Event StdOut  -----------------\n")
-		fmt.Println(statusEvent.StdOut)
-		fmt.Printf("\n-------------------------------------------------------------------------------\n")
+		str := "\n--------------------------- Ansible Task Status Event StdOut  -----------------\n"
+		str += fmt.Sprintf("\n----- %70s -----\n", fmt.Sprintf("%s %s/%s", u.GroupVersionKind(), u.GetName(), u.GetNamespace()))
+		str += fmt.Sprintf("\n%s\n", statusEvent.StdOut)
+		str += "\n-------------------------------------------------------------------------------\n"
+		fmt.Println(str)
```

Comment:
Recreating strings is inefficient. This should only be done once. GVK info should also be in the header to decrease the number of lines printed/confusion
```suggestion
		str := fmt.Sprintf("Ansible Task Status Event StdOut (%s, %s/%s)", u.GroupVersionKind(), u.GetName(), u.GetNamespace())
		fmt.Printf("\n----- %70s -----\n\n%s\n\n----------\n", str, statusEvent.StdOut)
```